src/components/greeting.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function Greeting(greeting) {
+  return (
+    <div>
+      <h1>Greetings</h1>
+      { [greeting].map((item) => (
+        <div key={item.id}>
+          <h2>{item.greeting}</h2>
+        </div>
+      ))}
+    </div>
```

Comment:
### Error displaying a random greeting
![image](https://github.com/Elhambasir/hello-react-front-end/assets/78251927/deeab62b-7060-4e7a-8aaf-16d7580c30dc)

- [ ] Since the API endpoint already selects a random greeting from your table, I don't think there is a need to use the `map` method.

- [ ] You can display the random greeting in your `Greeting` component directly to avoid errors.

- [ ] What do you think?🛠

Reply:
@abdulhamiid. I have made all requested changes. But there are some eslint errors that were causing an issue I disabled them for now. I checked them those were because of some dependencies but the code it self has no linter error at all.